application-services/custom/secrets/README.md

```diff
@@ -131,5 +97,12 @@ These tests use a collection of Postman requests, in *SecretsExample.postman_col
 
 4. View the service's logs to verify that the secrets were retrieved. We'll view the secrets in the application's console (in production, NEVER log your application's secrets. This is done in the example service to demonstrate the functionality).
 
-   1. Running in docker: `docker logs app-service-secrets-example`
-   2. Running natively: view the console
+   ```bash
```

Comment:
```suggestion
   ```console
```

Reply:
consule not a valid option

Reply:
```console
$ test console
```

Reply:
the original code:
`
#   ```console
   $ test console
   ```
`

Reply:
so to me it is an option at here :-)

Reply:
Using console only make text blue , wahhoo!
``` console
make build
./app-service
```

Reply:
added